src/lightning_app/core/flow.py

```diff
@@ -352,12 +356,11 @@ def schedule(
 
             from lightning_app import LightningFlow
 
-
             class Flow(LightningFlow):
+
                 def run(self):
                     if self.schedule("hourly"):
                         # run some code once every hour.
-                        print("run this every hour")
```

Comment:
This is not valid code if you remove this

Reply:
Done !